src/controllers/playback/videoosd.js

```diff
@@ -1122,6 +1122,7 @@ define(["playbackManager", "dom", "inputManager", "datetime", "itemHelper", "med
                     }
                     break;
                 case "k":
```

Comment:
Not directly related to this PR, but why is this K instead of something sensible like P ? (I get that JKL are homerow, next to each other and all, but for discovery by most users, K makes no sense)

Reply:
As on Youtube

Reply:
If is was going to be something else it would just be the arrow keys.

Reply:
Because `vim`?..

Reply:
I had no idea Youtube used that :p

Nevermind, then.